### DISCUSSION TITLE
Welcome to AHK_Minecraft_Tools Discussions!

### DISCUSSION THREAD

**Discussion by houdini101** [houdini101/AHK_Minecraft_Tools#2] [General Discussion]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jenni420** [comment]
I have problem your script is not working can u help me

**Reply by houdini101** [reply]
Yes, what can I help you with? Which part of the script is not working?
